resources/views/blueprints/index.blade.php

```diff
@@ -133,6 +133,24 @@
     </div>
     @endforeach
 
+    @foreach (Statamic\Facades\Blueprint::additionalBlueprints() as $title => $blueprints)
+        <h3 class="little-heading pl-0 mb-1">{{ $title }}</h3>
```

Comment:
If the namespace has been registered but there are no blueprints, there's a weird empty state.

![image](https://user-images.githubusercontent.com/105211/124806421-abddea80-df2a-11eb-9462-6a9f12902de0.png)


Could you prevent outputting anything for empty namespaces?

Reply:
Pushed a fix for that, i added a `->filter()` call to the collection returned to remove empty values